packages/flame/lib/src/components/position_component.dart

```diff
@@ -332,6 +348,11 @@ class PositionComponent extends Component
   /// The absolute center of the component.
   Vector2 get absoluteCenter => absolutePositionOfAnchor(Anchor.center);
 
+  /// Rotates the component to look at given target.
+  void lookAt(Vector2 target) {
+    angle = _nativeDirection.angleToSigned(target - absolutePosition);
```

Comment:
This should be possible to calculate without creating a new vector object

Reply:
Just to be clear, you are referring to new vector created because of `target - absolutePosition` and not `_nativeDirection`, right?

Reply:
Nvm. Got what you were saying. Changed the logic to not depend on additional vectors.